### ISSUE TITLE
Fiori 3 Design Review: Boarder List should have underline 

### ISSUE THREAD

**Issue by estelaxu3** [SAP/fundamental-styles#789]
In border list there should be an underline after it is selected for accessibility purpose: 

https://sap.github.io/fundamental-styles/components/list.html
